web-ui/src/components/guides/GuideLink.test.jsx

```diff
@@ -5,7 +5,15 @@ import { MemoryRouter } from 'react-router-dom';
 it('renders correctly', () => {
   snapshot(
     <MemoryRouter>
-      <GuideLink name="myFileName" />
+      <GuideLink id="some-id" name="myFileName" url="/pdfs/myFileName.pdf"/>
```

Comment:
Do `GuideLink` components need to have an id attribute?

Reply:
The id comes from the database https://github.com/objectcomputing/check-ins/pull/2522/files#diff-4be4111fc38431f8545a391dda60aca733603aceedbe7ae0ea91c376ca09c557R10 and is used as a `key` in the list values...is there a better way?